solidity/ecdsa/test/WalletRegistry.WalletOwner.test.ts

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable no-underscore-dangle */
+import { ethers, helpers } from "hardhat"
+import { expect } from "chai"
+
+import { params, walletRegistryFixture } from "./fixtures"
+import { submitRelayEntry } from "./utils/randomBeacon"
+import { signAndSubmitCorrectDkgResult } from "./utils/dkg"
+import ecdsaData from "./data/ecdsa"
+
+import type { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import type { DkgResult } from "./utils/dkg"
+import type { FakeContract } from "@defi-wonderland/smock"
+import type {
+  IWalletOwner,
+  WalletRegistry,
+  WalletRegistryStub,
+} from "../typechain"
+import type { ContractTransaction } from "ethers"
+
+const { mineBlocks } = helpers.time
+
+const { createSnapshot, restoreSnapshot } = helpers.snapshot
+
+describe("WalletRegistry - Wallet Owner", async () => {
+  const groupPublicKey: string = ethers.utils.hexValue(
+    ecdsaData.group1.publicKey
+  )
+  const groupPublicKeyHash: string = ethers.utils.keccak256(groupPublicKey)
+
+  let walletRegistry: WalletRegistryStub & WalletRegistry
+  let walletOwner: FakeContract<IWalletOwner>
+
+  before("load test fixture", async () => {
+    // eslint-disable-next-line @typescript-eslint/no-extra-semi
+    ;({ walletRegistry, walletOwner } = await walletRegistryFixture())
+  })
+
+  describe("approveDkgResult", async () => {
+    let dkgResult: DkgResult
+    let submitter: SignerWithAddress
+
+    before(async () => {
+      await createSnapshot()
+
+      await walletOwner.__ecdsaWalletCreatedCallback.reverts(
+        "wallet owner internal error"
+      )
```

Comment:
Why do we need it here?

Reply:
I have no idea how it got here 😕  I'll remove it!